script/test-template.sh

```diff
@@ -5,7 +5,7 @@ set -o pipefail  # Exit when any program execution in a pipeline breaks
 
 readonly APP="./bin/create-probot-app.js"
 readonly TEMPLATE=$1
-readonly TEST_FOLDER="./tmp-test/${TEMPLATE}"
+readonly TEST_FOLDER=$(mktemp -d -t cpa-XXXXXXXXXX)
```

Comment:
great use for `mktemp` :+1:

Reply:
I copied/pasted your suggestion 😬
BTW I was initially worried that might not be portable between macOS and Linux but it seems to work all right